lang/lang.go

```diff
@@ -169,6 +169,7 @@ func init() {
 	bundle = i18n.NewBundle(language.English)
 	bundle.RegisterUnmarshalFunc("json", json.Unmarshal)
 
+	translated = []language.Tag{language.Make("en")} // the first item in this list will be the fallback if none match
```

Comment:
This is kind-of assuming the app has supplied its own English translations, which to be fair is a pretty reasonable bet, but if not, it will lead to lots of "translation failure" logs since Fyne does supply an English translation and it will fail on all the strings unique to the app itself.

Do we want to track if the app developer has supplied their own english translation dictionary, and only if that's true, put english at the front?